CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased][unreleased]
 
+### Added
+- message-parse summary text in chatlist item summary #3476
```

Comment:
The CHANGELOG entry / PR title can maybe be improved as I'm not sure if I understand it. How about "Introduce text formatting for chatlist item summary" or similar?